paper/paper.md

```diff
@@ -0,0 +1,137 @@
+---
+title: 'TrixiParticles.jl: Particle-based multiphysics simulation in Julia'
+tags:
+  - Julia
+  - SPH
+  - particle-based
+  - multiphysics
+authors:
+  - name: Niklas S. Neher
+    orcid: 0009-0004-2472-0923
+    equal-contrib: true
+    affiliation: "1" # (Multiple affiliations must be quoted)
+  - name: Erik Faulhaber
+    orcid: 0000-0001-9788-5949
+    equal-contrib: true # (This is how you can denote equal contributions between multiple authors)
+    affiliation: 2
+  - name: Sven Berger
+    orcid: 0000-0001-6083-7038
+    equal-contrib: true # (This is how you can denote equal contributions between multiple authors)
+    affiliation: 3
+  - name: Michael Schlottke-Lakemper
+    orcid: 0000-0002-3195-2536
+    affiliation: 1 # TODO: @Michael: Stuttgart or Augsburg
+  - name: Gregor J. Gassner
+    orcid: 0000-0002-1752-1158
+    affiliation: 2
+affiliations:
+ - name: High-Performance Computing Center Stuttgart, University of Stuttgart, Germany
+   index: 1
+ - name: Department of Mathematics and Computer Science, University of Cologne, Germany
+   index: 2
+ - name: Institute of Surface Science, Helmholtz-Zentrum hereon, Germany
+   index: 3
+date: 27 March 2024 # TODO
+bibliography: paper.bib
+
+# Optional fields if submitting to a AAS journal too, see this blog post:
+# https://blog.joss.theoj.org/2018/12/a-new-collaboration-with-aas-publishing
+aas-doi: 10.3847/xxxxx <- update this with the DOI from AAS once you know it.
+aas-journal: Astrophysical Journal <- The name of the AAS journal.
+---
+
+# Summary
```

Comment:
Is not really a summary but an introduction at the moment. So this needs to be reformulated to go: 1 Paragraph introduction, 2nd/3rd paragraph summary of the rest of the paper.